pyomo/contrib/doe/doe.py

```diff
@@ -481,15 +529,25 @@ def _sequential_finite(self, read_output, extract_single_model, store_output):
 
     def _direct_kaug(self):
         # create model
-        mod = self.create_model(model_option=ModelOptionLib.parmest)
+        if self._original_create_model_interface:
+            mod = self.create_model(model_option=ModelOptionLib.parmest, **self.args)
+        else:
+            mod = self.create_model(**self.args)
 
         # discretize if needed
-        if self.discretize_model:
+        if self.discretize_model is not None:
             mod = self.discretize_model(mod, block=False)
```

Comment:
I know this is from legacy, but are there ever checks to make sure block is a keyword arg?